README.md

```diff
@@ -606,8 +606,19 @@ crud.disable_stats()
 crud.reset_stats()
 ```
 
-Format is as follows.
+If [`metrics`](https://github.com/tarantool/metrics) `0.9.0` or greater
+found, metrics collectors will be used by default to store statistics
+instead of local collectors. You can manually choose driver if needed.
 ```lua
+-- Use metrics collectors.
+crud.enable_stats({ driver = 'metrics' })
```

Comment:
Why do we need to enable and disable statistics aggregation?  I suppose it is due to overhead when metrics is used. If so, it would be nice to mention it here.

Reply:
There is a

> Performance overhead is 3-5% in case of `local` driver and
> 10-20% in case of `metrics` driver.

comment added in performance test commit. It is right below drivers descriptions.